tests/containers/workbanches/jupyterlab/test_mysql.py

```
@@ -0,0 +1,45 @@
+    result = cursor.fetchone()
+    if result == (1,):
+        print("MySQL connection successful!")
+    else:
+        print("MySQL connection failed!")
+    cnx.close()
+except Exception as e:
+    print(f"An error occurred: {{e}}")
+"""
+
+        # Execute the python script inside the workbench container
+        exit_code, (stdout, stderr) = workbench_container.exec_run(
```

Ruff (0.12.2)

Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)